libhb/qsv_common.c

```diff
@@ -2291,8 +2312,6 @@ void hb_qsv_force_workarounds()
 #undef FORCE_WORKAROUNDS
 }
 
-AVBufferRef *enc_hw_frames_ctx = NULL;
-extern EncQSVFramesContext hb_enc_qsv_frames_ctx;
 AVBufferRef *hb_hw_device_ctx = NULL;
```

Comment:
Not part of this PR, but another global being used for encoding context.  It looks like this can also be put in job->qsv.ctx and retrieved from (hb_job_t*)AVCodecContext.opaque where it is used.

I suggest adding a TODO for future cleanup